frontend/packages/dev-console/src/components/hpa/hpa-utils.ts

```diff
@@ -29,17 +30,23 @@ export const getLimitWarning = (resource: K8sResourceKind): string => {
   const limits = getResourceLimitsData(resource);
 
   if (!limits) {
-    return `CPU and memory resource limits must be set if you want to use CPU and memory utilization. \
-    The ${HorizontalPodAutoscalerModel.label} will not have CPU or memory metrics until resource limits are set.`;
+    return i18next.t(
+      'public~CPU and memory resource limits must be set if you want to use CPU and memory utilization. The {{label}} will not have CPU or memory metrics until resource limits are set.',
+      { label: HorizontalPodAutoscalerModel.label },
```

Comment:
Just inline "HorizontalPodAutoscaler" in the message in case we want to translate the kind. Same for the instances below.